app/users/views.py

```diff
@@ -2,12 +2,13 @@
 from rest_framework import generics
 from rest_framework.response import Response
 from rest_framework.views import APIView
+from rest_framework import viewsets
 
 from .serializers import UserSerializer, UserProfileSerializer
 from .models import User, UserProfile
 
 
-class UserList(generics.ListAPIView):
+class UserList(generics.ListAPIView, viewsets.ModelViewSet):
```

Comment:
What is the point of inheriting from both ListAPIView and ModelViewSet? Only ModelViewSet should work.